layers/+email/mu4e/packages.el

```diff
@@ -36,6 +38,9 @@
       (add-to-list 'mu4e-view-actions
                    '("View in browser" . mu4e-action-view-in-browser) t)
 
+      (when (configuration-layer/layer-usedp 'org)
+        (require 'org-mu4e))
```

Comment:
I don't understand why org-mu4e has to be two required, this does not make sense to me, Only one should be necessary and this is the second one, if this is not the case then `org-mu4e` is missing some `(require 'org)` which does not make sense.